api/src/main/java/com/trainlab/controller/FrontendDataController.java

```diff
@@ -50,20 +32,5 @@ public class FrontendDataController {
                     )
             }
     )
-    @GetMapping("/pages/{range}")
-    public ResponseEntity<Map<String, String>> getMainPageData(@PathVariable int range) {
-
-        List<FrontendData> mainPageDataList = frontendDataRepository.findDataByRange(range);
-
-        if (mainPageDataList.isEmpty()) {
-            return new ResponseEntity<>(HttpStatus.NOT_FOUND);
-        }
-
-        Map<String, String> mainPageDataMap = new HashMap<>();
-        for (FrontendData data : mainPageDataList) {
-            mainPageDataMap.put(Float.toString(data.getFrontId()), data.getText());
-        }
-
-        return new ResponseEntity<>(mainPageDataMap, HttpStatus.OK);
-    }
+    ResponseEntity<Map<String, String>> getMainPageData(@PathVariable int page);
```

Comment:
Вместо примитивов лучше использовать обертки

```suggestion
    ResponseEntity<Map<String, String>> getMainPageData(Integer page);
```